src/controllers/user.js

```diff
@@ -115,3 +115,21 @@ export const getStudents = async (req, res) => {
     return sendMessageResponse(res, 500, 'Server Not Found')
   }
 }
+
+export const addUserToCohort = async (req, res) => {
+  const { student_Id: studentId } = JSON.parse(req.body.body)
```

Comment:
Should this be `req.body.body`? That would mean a request would look like: 
```
{
   body: {
      student_Id: 1
   }
}
```

Reply:
Not sure why but `req.body` doesn't seem to work whereas `req.body.body` does.